tutorials/install-distro.md

```diff
@@ -0,0 +1,72 @@
+| 1.17 | x          | x        | x        | x               | x        | x               | x              | x         | x          | x             | x             | x             | x             |
+| 1.16 | x          | x        | x        | x               | x        | x               | x              | x         | x          | x             | x             | x             | x             |
+
+To install, choose a supported version for your operating system, and export it as a variable, like so:
+`export VERSION=1.18`
+
+
+## Installation Instructions
+
+### Fedora 31 or later
+```shell
+sudo dnf module enable cri-o:$VERSION
+sudo dnf install cri-o
+```
+
+### openSUSE:
+```shell
+sudo zypper install cri-o
+```
+
+
+### Other yum based operating systems
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
+
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Centos 8         | `CentOS_8`        |
+| Centos 8 Stream  | `CentOS_8_Stream` |
+| Centos 7         | `CentOS_7`        |
+
+
+And then run the following as root:
+```shell
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable.repo https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/devel:kubic:libcontainers:stable.repo
+curl -L -o /etc/yum.repos.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/devel:kubic:libcontainers:stable:cri-o:$VERSION.repo
+yum install cri-o
+```
+
+
+### Deb based operating systems
+
+Note: this tutorial assumes you have curl and gnupg installed
+
+To install on the following operating systems, set the environment variable $OS as the appropriate field in the following table:
+| Operating system | $OS               |
+| ---------------- | ----------------- |
+| Debian Unstable  | `Debian_Unstable` |
+| Debian Testing   | `Debian_Testing`  |
+| Ubuntu 20.04     | `xUbuntu_20.04`   |
+| Ubuntu 19.10     | `xUbuntu_19.10`   |
+| Ubuntu 19.04     | `xUbuntu_19.04`   |
+| Ubuntu 18.04     | `xUbuntu_18.04`   |
+
+And then run the following as root:
+```shell
+echo "deb https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list
+echo "deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable:/cri-o:/$VERSION/$OS/ /" > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable:cri-o:$VERSION.list
+
+curl -L https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable:cri-o:$VERSION/$OS/Release.key | apt-key add -
+curl -L https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/$OS/Release.key | apt-key add -
+
+apt-get update
+apt-get install cri-o
```

Comment:
Not sure if it is already somewhere, but maybe we should add a `install from source` steps?

Reply:
yeah it's in tutorials/setup.md. I suppose I could combine the two into one unified install.md?

Reply:
fixed!